### ISSUE TITLE
[Catalog] App Bar "Start Demo" has no accessible elements below the AppBar

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4337]
## Details
The "Start Demo" example in App Bar has no content in the table view below the app bar. This makes it difficult for users to determine how the App Bar will interact with on-screen content.

## Reproduction steps
1. Open the MDCCatalog app
1. Navigate to App Bar &rarr; Start Demo
1. Using VoiceOver, navigate the screen elements

## Expected behavior
The table view below the app bar has accessible content in at least one cell

## Actual behavior
There appears to be content (it's visible) but VoiceOver cannot interact with it.

## Platform (please complete the following information)
iPhone 5c/iOS 10.3.3

---
#### Internal data
- Associated internal bug: [b/117178314](http://b/117178314)

**Comment by material-mirror** [comment]
The internal issue [b/117178314](http://b/117178314) is now closed. This issue is being closed as a result.
